app/controllers/v0/institutions_controller.rb

```diff
@@ -40,6 +40,22 @@ def show
              meta: { version: @version }, links: @links
     end
 
+    # GET /v0/institituons/20005123/children
+    def children
+      children = Institution.version(@version[:number]).where(parent_facility_code_id: params[:id])
```

Comment:
You have the query divided up between this line and line 53.
You can move it all here:
```suggestion
      children = Institution.version(@version[:number]).where(parent_facility_code_id: params[:id]).order(:institution).page(params[:page])
```

I believe the `children.count` will still return the full count;  the `page` method does not affect the count.

Reply:
Also, following @annaswims's comment [here](https://github.com/department-of-veterans-affairs/gibct-data-service/pull/376/files#r298224965), it would be more performant if you wrote it as
```suggestion
      children = Institution.where(version: @version[:number], parent_facility_code_id: params[:id]).order(:institution).page(params[:page])
```
and added a composite index on `version` and `parent_facility_code_id`

Reply:
This one still needs addressing.

Reply:
Thanks!